### PR TITLE
Redesign v2

### PR DESCRIPTION
**Pull request by Adrian-St**
Adds hover to buttons.
Refactor footer in unauhorized App. 
Small fixes.

### CODE REVIEW

File: src/components/UnauthorizedApp.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  return (
+    <Fragment>
+      <div className="UnauthorizedApp">
+        <Switch>
+          <Route path="/login" component={LoginForm} />
+          <Route path="/register" component={RegisterForm} />
+          <Route path="/forgot-password" component={PasswordRequestForm} />
+          <Route path="/reset-password" component={ResetPasswordForm} />
+          <Route exact path="/" component={LandingPage} />
+          <AuthenticatedRoute path="*" component={AuthorizedApp} />
```
Comment (arne-z) [inline src/components/UnauthorizedApp.js]: If more than one of these switch statements is true, where does it lead? Eg. if I am trying to visit /login while I am logged in, will the `          <Route path="/login" component={LoginForm} />
` or the `          <AuthenticatedRoute path="*" component={AuthorizedApp} />
` rule take precedence?

File: src/components/landing_page/LandingPage.scss [hunk 2]
```diff
@@ -29,8 +28,8 @@
 
 .Particles {
   width: 100%;
-  height: 100%;
-  position: fixed;
+  height: calc(100% - var(--footer-height));
```
Comment (arne-z) [inline src/components/landing_page/LandingPage.scss]: The way the footer was done on the landing page previously (transparent background) was a specific request by @luisebenkert. It looks like you are re-adding the dark border at the bottom of the landing page here, by deleting the <LandingPageFooter /> and using the Unauthorized Footer here instead.

Comment (Adrian-St) [reply src/components/landing_page/LandingPage.scss]: Yeah, but the footer is always hidden and you can only see it when scrolling down.

Comment (arne-z) [reply src/components/landing_page/LandingPage.scss]: That sounds good then 👍

**Pull request merged**